hosts/testagent-dev/configuration.nix

```diff
@@ -140,10 +140,23 @@ in
   };
 
   # Enable Acroname USB Smart switch, as well as LXA USB-SD-Mux support.
-  services.udev.packages = [
-    brainstem
-    pkgs.usbsdmux
-  ];
+  #services.udev.packages = [
```

Comment:
Remove this block completely instead of only commenting out.

Reply:
fixed